src/ralph/data_center/models/physical.py

```diff
@@ -214,6 +214,8 @@ def get_free_u(self):
 
         def fill_u_list(objects, height_of_device=lambda obj: 1):
             for obj in objects:
+                if not obj[0]:
```

Comment:
the question here is if object with position 0 should (or could) have non-zero height: https://github.com/allegro/ralph/blob/ng/src/ralph/data_center/models/physical.py#L333 ? Anyone know something about it?